CHANGES.rst

```diff
@@ -57,6 +57,15 @@ Imviz
   linked to the data collection, resulting in Imviz unusable until
   the data are re-linked manually. [#1365]
 
+- Fixed a bug where coordinates display erroneously showing info from
+  the reference image even when it is not visible. [#1392]
```

Comment:
This seems to now account for `layer.visible`, but does it need to also account for `layer.state.bitmap_visible` and/or `layer.state.contour_visible`?

Reply:
Hmm blink only sets `.visible`. I wasn't even aware of the other two attributes. What do they control?

Reply:
Whether the image and contours are visible.  So in order to see an image, both `layer.visible` and `layer.state.bitmap_visible` must be True (iirc).

Reply:
I never had to set `.bitmap_visible` for it to show. Is something not working for you?

Reply:
If you leave the top layer's visibility on, but disable the `bitmap_visible` (either through the API or through the plot options), then the coordinates display will still use that layer, even though the user is not seeing it.  You could argue that this is expected behavior since the layer as a whole is still enabled and that this will be "addressed" if/when we add an indication of the label of the "top/active" layer.